src/postcode-regexes.ts

```diff
@@ -172,5 +172,27 @@ export const POSTCODE_REGEXES: Map<string, RegExp> = new Map([
   [CountryCode.XK, /^\d{5}$/],
   [CountryCode.YT, /^976\d{2}$/],
   [CountryCode.PE, /^\d{5}$/],
+  [CountryCode.GI, /^GX11 1AA/],
+  [CountryCode.BT, /^\d{5}/],
+  [CountryCode.AL, /^\d{4}/],
+  [CountryCode.CU, /^\d{5}/],
+  [CountryCode.UM, /^96898/],
+  [CountryCode.AI, /^(?:AI-)?2640/],
+  [CountryCode.AF, /^\d{4}/],
+  [CountryCode.SD, /^\d{5}/],
+  [CountryCode.VC, /^VC\d{4}/],
+  [CountryCode.TA, /^TDCU 1ZZ/],
+  [CountryCode.NA, /^\d{5}/],
+  [CountryCode.EH, /^\d{5}/],
+  [CountryCode.BL, /^9[78][01]\d{2}/],
+  [CountryCode.TZ, /^\d{4,5}/],
+  [CountryCode.AC, /^ASCN 1ZZ/],
+  [CountryCode.VG, /^VG\d{4}/],
+  [CountryCode.MZ, /^\d{4}/],
+  [CountryCode.MF, /^9[78][01]\d{2}/],
+  [CountryCode.MM, /^\d{5}/],
+  [CountryCode.SV, /^CP [1-3][1-7][0-2]\d/],
+  [CountryCode.CO, /^\d{6}/],
```

Comment:
Also for Columbia, I think https://github.com/melwynfurtado/postcode-validator/pull/202/files#diff-6bee2dbe56eb6b4193c2d9f9758bf594a4062159b7f4a9c8df2f0a746c0a471cR176 is more accurate. So lets merge this one first and then add the missing ones. Thanks.